application/src/main/java/org/togetherjava/tjbot/commands/help/HelpThreadCommand.java

```diff
@@ -204,6 +205,18 @@ private void closeThread(SlashCommandInteractionEvent event, ThreadChannel helpT
         event.replyEmbeds(embed).flatMap(any -> helpThread.getManager().setArchived(true)).queue();
     }
 
+    private void bookmarkThread(SlashCommandInteractionEvent event, ThreadChannel helpThread) {
+        event.getUser()
+            .openPrivateChannel()
+            .flatMap(channel -> channel.sendMessage(String.format("<#%s>", helpThread.getIdLong())))
```

Comment:
the message is lacking any sort of context. Maybe add some:
```java
You bookmarked the following thread:
* #foobar
```

Reply:
will that look good after a few bookmarks

Reply:
> will that look good after a few bookmarks

Not really, but keep in mind that bot DMs a lot of different things to the user. Having just random thread links in-between other messages doesn't sound like a good UX.

Reply:
> Having just random thread links in-between other messages doesn't sound like a good UX.

yes, i'd like to have something in one single like not multi-line so doing that zab said but in single like

Reply:
That is very much unrelated to what marko said.

Marko is talking about messages in between, not multi-line messages